pdfarranger/exporter.py

```diff
@@ -399,6 +400,7 @@ def num_pages(filepath):
 
 
 def generate_booklet(pdfqueue, tmp_dir, pages):
+    pre_pike_2_7 = version.parse(pikepdf.__version__) < version.Version('2.7.0')
```

Comment:
Why not use parse or Version constructor on both sides? What is the difference actually?

Reply:
I believe there is no real difference. Using both is a stylistic preference - it feels more natural to me for a literal and parse for an unknown version, but if you want me to use one or the other I don't have a problem with changing it.